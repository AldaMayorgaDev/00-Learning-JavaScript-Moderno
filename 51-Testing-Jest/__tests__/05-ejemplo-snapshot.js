/* Testing con JEST usando snapshots*/
/* Los snapshots sin datos que se alamacenan en un string, se crear en una carpeta, aparte y sobre ella podemos comparar si es el mismo dato, es decir, en este caso si es el mismo cliente o el mismo dato ,*/
const cliente = {
    nombre: 'Aldahir',
    balance: 500,
    tipo: 'Premium'
}

describe('Testing al objeto Cliente', () => {
    test('Es el mismo objeto Aldahir', () => { 
        expect(cliente).toMatchSnapshot();
     })

})