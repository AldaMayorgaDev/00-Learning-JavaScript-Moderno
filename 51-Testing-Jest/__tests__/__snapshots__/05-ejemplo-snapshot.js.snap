// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al objeto Cliente Es el mismo objeto Aldahir 1`] = `
{
  "balance": 500,
  "nombre": "Aldahir",
  "tipo": "Premium",
}
`;
